services/dynamic-sidecar/Makefile

```diff
@@ -0,0 +1,20 @@
+include ../../scripts/common.Makefile
+include ../../scripts/common-service.Makefile
+
+APP_NAME := $(notdir $(CURDIR))
+
+.DEFAULT_GOAL := help
+
+
+.PHONY: codestyle
+codestyle: ## enforces codestyle and runs pylint and mypy
+	isort setup.py src/simcore_service_dynamic_sidecar tests
+	black src/simcore_service_dynamic_sidecar tests/
+	pylint --rcfile=../../.pylintrc src/simcore_service_dynamic_sidecar tests/
+	mypy src/simcore_service_dynamic_sidecar tests/ --ignore-missing-imports
```

Comment:
same as above. it would be nice to have everywhere? not possible to set that in the base makefile?

Reply:
see comment above